### PR TITLE
Remove outputDir and dataDir in config file

### PR DESCRIPTION
**Pull request by SparkSnail**
https://github.com/microsoft/nni/issues/1342

### CODE REVIEW

File: docs/en_US/TrainingService/PaiMode.md [hunk 1]
```diff
@@ -32,8 +32,6 @@ trial:
   cpuNum: 1
   memoryMB: 8196
   image: msranni/nni:latest
```
Comment (xuehui1991) [inline docs/en_US/TrainingService/PaiMode.md]: so I want to output my models, where is the output path?

Comment (QuanluZhang) [reply docs/en_US/TrainingService/PaiMode.md]: PAI does not have `dataDir` and `outputDir` in their doc anymore, so we remove these two path accordingly. In other training modes, users use `OUTPUTDIR` in trial code to output the data they want, @SparkSnail could you check the content of `OUTPUTDIR` in PAI mode?

File: src/nni_manager/training_service/pai/paiTrainingService.ts [hunk 2]
```diff
@@ -497,12 +451,8 @@ class PAITrainingService implements TrainingService {
             trialJobDetail.paiJobName,
             // Docker image
             this.paiTrialConfig.image,
-            // dataDir
-            this.paiTrialConfig.dataDir,
-            // outputDir
-            this.paiTrialConfig.outputDir,
```
Comment (QuanluZhang) [inline src/nni_manager/training_service/pai/paiTrainingService.ts]: so in the previous version, `outputDir` in nni config is also `outputDir` in pai job config?

File: tools/nni_cmd/launcher_utils.py [hunk 3]
```diff
@@ -253,6 +253,12 @@ def validate_pai_trial_conifg(experiment_config):
         experiment_config['trial']['shmMB'] > experiment_config['trial']['memoryMB']:
             print_error('shmMB should be no more than memoryMB!')
             exit(1)
+        #backward compatibility
+        warning_information = '{0} is not supported in NNI anymore, please remove the field in config file!'
```
Comment (QuanluZhang) [inline tools/nni_cmd/launcher_utils.py]: could add a little more information, for example, "please refer to the link for the practices of how to get data and output model in trial code".

File: docs/en_US/TrainingService/PaiMode.md (hunk 1 above)
Comment (xuehui1991) [reply docs/en_US/TrainingService/PaiMode.md]: any doc to tell?

Comment (SparkSnail) [reply docs/en_US/TrainingService/PaiMode.md]: Users could use `PAI_OUTPUT_DIR` to get the output directory of PAI, the defaulet value is $PAI_DEFAULT_FS_URI/Output/$jobName, refer https://github.com/microsoft/pai/blob/b2324866d0280a2d22958717ea6025740f71b9f0/docs/job_tutorial.md

File: src/nni_manager/training_service/pai/paiTrainingService.ts (hunk 2 above)
Comment (SparkSnail) [reply src/nni_manager/training_service/pai/paiTrainingService.ts]: yes

File: tools/nni_cmd/launcher_utils.py (hunk 3 above)
Comment (SparkSnail) [reply tools/nni_cmd/launcher_utils.py]: updated.

File: docs/en_US/TrainingService/PaiMode.md [hunk 4]
```diff
@@ -80,9 +74,10 @@ And you will be redirected to HDFS web portal to browse the output files of that
 
 You can see there're three fils in output folder: stderr, stdout, and trial.log
 
-If you also want to save trial's other output into HDFS, like model files, you can use environment variable `NNI_OUTPUT_DIR` in your trial code to save your own output files, and NNI SDK will copy all the files in `NNI_OUTPUT_DIR` from trial's container to HDFS.
+## data management
+If your training data is not to large, the could be put into codeDir, and nni will upload the data to hdfs, or you could build your own docker image with the data. If you have large dataset, it's not appropriate to put the data in codeDir, and you could follow the [guidance](https://github.com/microsoft/pai/blob/master/docs/user/storage.md) to mount the data folder in container.
```
Comment (chicm-ms) [inline docs/en_US/TrainingService/PaiMode.md]: typo:   1.  to,  2. the

File: docs/en_US/TrainingService/PaiMode.md [hunk 5]
```diff
@@ -92,4 +87,6 @@ Check policy:
 3. Note that the version check feature only check first two digits of version.For example, NNIManager v0.6.1 could use trialKeeper v0.6 or trialKeeper v0.6.2, but could not use trialKeeper v0.5.1 or trialKeeper v0.7.
 
 If you could not run your experiment and want to know if it is caused by version check, you could check your webUI, and there will be an error message about version check.
-![](../../img/version_check.png)
\ No newline at end of file
+![](../../img/version_check.png)
+
+Any problems when using NNI in pai mode, please create issues on [NNI github repo](https://github.com/Microsoft/nni).
```
Comment (chicm-ms) [inline docs/en_US/TrainingService/PaiMode.md]: Suggest to remove this sentence,  the process to create issue is identical for all nni component,  no difference.

File: docs/en_US/TrainingService/PaiMode.md (hunk 4 above)
Comment (SparkSnail) [reply docs/en_US/TrainingService/PaiMode.md]: fixed.

File: docs/en_US/TrainingService/PaiMode.md (hunk 5 above)
Comment (SparkSnail) [reply docs/en_US/TrainingService/PaiMode.md]: fixed.

**Pull request merged**